specification/loadtestservice/LoadTestService.Management/main.tsp

```diff
@@ -0,0 +1,29 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "./routes.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.ResourceManager;
+
+@armProviderNamespace
+@service({
+  title: "LoadTest Client",
+})
+@doc("LoadTest client provides access to LoadTest Resource and it's status operations.")
+@versioned(APIVersions)
+@armCommonTypesVersion(Azure.ResourceManager.CommonTypes.Versions.v5)
```

Comment:
This is causing your swagger to use the v5 common types, where before it used the v3 common types.  This may be an sdk breaking change.

Reply:
As suggested by @allenjzhang in this [comment](https://github.com/Azure/azure-rest-api-specs/pull/27565#discussion_r1520465367), there is no major change between v3 to v5. 
Also, the typespec compiler is throwing following warning when I use v3 common types. 
`warning @azure-tools/typespec-azure-resource-manager/arm-common-types-incompatible-version: No ARM common-types version for this type satisfies the expected version v3.  This type only supports the following version(s): v5, v4`
Shall I still use v3 or can we go ahead with v5?

Reply:
Using v5 to prevent Typespec validation failure